build.go

```diff
@@ -0,0 +1,90 @@
+package miniconda
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/paketo-buildpacks/packit"
+	"github.com/paketo-buildpacks/packit/postal"
+)
+
+//go:generate faux --interface DependencyManager --output fakes/dependency_manager.go
+type DependencyManager interface {
+	Resolve(path, id, version, stack string) (postal.Dependency, error)
+	Deliver(dependency postal.Dependency, cnbPath, destinationPath, platformPath string) error
+}
+
+//go:generate faux --interface EntryResolver --output fakes/entry_resolver.go
+type EntryResolver interface {
+	MergeLayerTypes(name string, entries []packit.BuildpackPlanEntry) (launch, build bool)
+}
+
+//go:generate faux --interface Runner --output fakes/runner.go
+type Runner interface {
+	Run(runPath, layerPath string) error
+}
+
+func Build(entryResolver EntryResolver, dependencyManager DependencyManager, runner Runner) packit.BuildFunc {
+	return func(context packit.BuildContext) (packit.BuildResult, error) {
+
+		dependency, err := dependencyManager.Resolve(filepath.Join(context.CNBPath, "buildpack.toml"), "miniconda3", "*", context.Stack)
+		if err != nil {
+			return packit.BuildResult{}, err
+		}
+
+		bom := packit.BOMEntry{
```

Comment:
Packit has this released https://github.com/paketo-buildpacks/packit/blob/v0.10.0/postal/service.go#L185